src/PrestaShopBundle/Form/Admin/Type/SwitchType.php

```diff
@@ -52,6 +52,7 @@ public function configureOptions(OptionsResolver $resolver)
             'multiple' => false,
             'expanded' => false,
             'disabled' => false,
+            'required' => false,
             'choice_translation_domain' => self::TRANS_DOMAIN,
```

Comment:
This was already done as part of this PR https://github.com/PrestaShop/PrestaShop/pull/21672 then I accidentaly managed to remove it(probably during the merge or something) as part of this PR https://github.com/PrestaShop/PrestaShop/pull/21245 . 

So here I am adding it again :disappointed:

Reply:
It happens. No worries. 👍